test/uuid-enforcement.ts

```diff
@@ -0,0 +1,79 @@
+import { launchAndConnect, testVersion, TEST_TIMEOUT, cleanOpenRuntimes, getRuntimeProcessInfo } from './multi-runtime-utils';
+import * as assert from 'assert';
+import { conn } from './connect';
+import * as http from 'http';
+import { Fin } from '../src/main';
+/* tslint:disable:no-invalid-this no-function-expression insecure-random mocha-no-side-effect-code no-empty max-func-body-length */
+let uuidCount = 0;
+let realmCount = 0;
+//tslint:disable-next-line
+const manifest = {
+    startup_app: { uuid: '', name: '' },
+    runtime: { version: testVersion, securityRealm: '' }
+};
+let fin: Fin;
+//tslint:disable-next-line
+const updateManifest = (uuid: string, realm = 'uuid-enforcement-realm-' + realmCount++) => {
+   manifest.startup_app.uuid = uuid;
+   manifest.startup_app.name = uuid;
+   manifest.runtime.securityRealm = realm;
+};
+const server = http.createServer((req, res) => res.end(JSON.stringify(manifest)));
+const serverReady = new Promise((y, n) => {
+    server.listen(5000, (e: any) => e ? n(e) : y());
+});
+//tslint:disable-next-line
+const newUuid = () => `duplicatedUuid${uuidCount++}`;
+
+function makeTest<T>(func1: (uuid: string) => Promise<T>, func2: (uuid: string, ret?: T) => Promise<any>) {
+    return async function () {
+        const uuid = newUuid();
+        const ret = await func1(uuid);
+        let secondFailed = false;
+        try {
+            await func2(uuid, ret);
+        } catch (e) {
+            secondFailed = true;
+        }
+        assert.ok(secondFailed);
+    };
+}
+const createFromManifest = async (uuid: string) => {
+    await serverReady;
+    updateManifest(uuid);
+    const app = await fin.Application.createFromManifest('http://localhost:5000');
+    return app.run();
```

Comment:
Since this method is deprecated, use startFromManifest instead.
question: you only pass url without json file, don't you get parsing error?